### PR TITLE
Update Release.toml and CHANGELOG for 1.4.0

### PR DESCRIPTION
**Pull request by cbgbt**
**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by zmrow — APPROVED** [review]
:jigsaw:

**Review by tjkirch — APPROVED** [review]

**Review by jpculp — APPROVED** [review]

**Review by webern — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
+# v1.4.0 (2021-11-12)
+
+## OS Changes
+
+* Add 'apiclient exec' for running commands in host containers ([#1802], [#1790])
+* Improve boot performance by 5-8 seconds ([#1809])
```
Comment (bcressey) [inline CHANGELOG.md]: Most of the improvement is specific to the `aws-k8s` variants, which I should've been more specific about in the PR. `aws-ecs-1` didn't take the hit from copying CNI plugins or having kubelet wait for the docker daemon, and won't see as much benefit.

```suggestion
* Improve boot performance for recent Kubernetes variants([#1809])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+* Allow wildcard container registry mirrors ([#1791])
+* Increase wicked `ifup` timeout from 30s to 300s ([#1800])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Wait up to 300s for a DHCP lease at boot ([#1800])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
+# v1.4.0 (2021-11-12)
+
+## OS Changes
+
+* Add 'apiclient exec' for running commands in host containers ([#1802], [#1790])
+* Improve boot performance by 5-8 seconds ([#1809])
+* Allow wildcard container registry mirrors ([#1791])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Add support for wildcard container registry mirrors ([#1791])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+* Increase wicked `ifup` timeout from 30s to 300s ([#1800])
+* Retry if fetching the IMDS session token fails ([#1801])
+* Add ECR account IDs for pulling host containers in GovCloud ([#1793])
+* Filter sensitive API settings from `logdog` dump ([#1777])
+* Fix pluto in k8s standalone mode ([#1783])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Fix kubelet standalone mode ([#1783])
```

**Comment by cbgbt** [comment]
^ Makes changes per @bcressey's suggestions.

**Review by tjkirch — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**